Update Networking intro and move Multiplayer Gem topics underneath (#2051)
o3de/o3de.org: content/docs/user-guide/networking/settings.md

---
linktitle: Network and Multiplayer Settings
title: AzNetworking and Multiplayer Gem Settings

> description: A reference for console variables and other settings that can be used on clients and servers to configure AzNetworking and the Multiplayer Gem.

Reviewer: ```suggestion
description: A reference for console variables and other settings that you can use on clients and servers to configure `AzNetworking` and the Multiplayer Gem.
```

weight: 300
---

## Overview

This page documents [console variables](/docs/user-guide/appendix/cvars/) and other settings that can control Networking and Multiplayer behavior.

## Networking commands

> The following [console functor](/docs/user-guide/programming/az-console/#console-functors-cfuncs) commands should be used to control the connection of clients to servers for networked games or simulations. 

Reviewer: ```suggestion
Use the following [console functor (cfunc)](/docs/user-guide/programming/az-console/#console-functors-cfuncs) commands to control the connection of clients to servers for networked games or simulations. 
```
